vsts/windows_csharp.cmd

```diff
@@ -4,24 +4,15 @@
 setlocal
 
 set build-root=%~dp0..
-rem // resolve to fully qualified path
+@rem resolve to fully qualified path
 for %%i in ("%build-root%") do set build-root=%%~fi
 
 cd %build-root%
 
-rem Build Debug + run UTs
-call build.cmd -clean -configuration Debug
-if errorlevel 1 goto :eof
-
+@rem Build Debug + run UTs and select E2E
 taskkill /F /IM Simulator.exe
 start /D .\vsts\TpmSimulator Simulator.exe
-
-rem Build Release + run E2E Tests
-rem If the build is Delay Signed, this requires strong name validation disabled for our key:
-rem (As Administrator:)  sn -Vr *,31bf3856ad364e35
-rem Re-enable strong name validation for this key using:
-rem (As Administrator:)  sn -Vx
-call build.cmd -clean -configuration Release -e2etests
+call build.cmd -clean -build -configuration Debug -prtests
```

Comment:
we should discuss the impact of modifying this on the release pipeline

Reply:
shouldn't this have -unittests as well?

Reply:
> shouldn't this have -unittests as well?

default is unit test+e2e

Reply:
The release script calls into build.cmd directly; so modifying this file should have no impact.